src/test/java/org/sagebionetworks/bridge/services/AuthenticationServiceMockTest.java

```diff
@@ -1284,6 +1294,12 @@ public void getSessionFromAccount() {
         assertEquals(session.getConsentStatuses(), CONSENTED_STATUS_MAP);
         
         verify(accountSecretDao).createSecret(AccountSecretType.REAUTH, USER_ID, REAUTH_TOKEN);
+        
+        RequestContext retValue = RequestContext.updateFromSession(session, null);
```

Comment:
When I do this in Eclipse, it gives me a warning: "The static method updateFromSession(UserSession, SponsorService) from the type RequestContext should be accessed in a static way." (Findbugs does not complain). So that's why I called it as a static method.